src/pages/index.js

```diff
@@ -177,32 +184,25 @@ const IndexPage = () => (
                 for accelerated learning.
               </Li>
             </Ul>
-            <P>
-              <Link to="/curriculum">
-                Checkout the curriculum
-              </Link>
-            </P>
-            <SectionButtonRow>
-              <Col sm={4}>
+            {/*<SectionButtonRow>
```

Comment:
@shakersinthedark it's better not to leave commented code. Please remove it or add a comment with the reason why that code stays commented there.